src/Microsoft.ML.TorchSharp/AutoFormerV2/Anchors.cs

```diff
@@ -0,0 +1,153 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using TorchSharp;
+using static TorchSharp.torch;
+using static TorchSharp.torch.nn;
+
+namespace Microsoft.ML.TorchSharp.AutoFormerV2
+{
+    /// <summary>
+    /// Anchor boxes are a set of predefined bounding boxes of a certain height and width, whose location and size can be adjusted by the regression head of model.
+    /// </summary>
+    public class Anchors : Module<Tensor, Tensor>
```

Comment:
How does an Anchor differ from a bounding box? Why don't we just name these BoundingBox?

Just trying to learn :)

Reply:
I am not sure about this case, but `Anchor` is the correct name when doing image cropping. In addition to the cropping box, you need to know if you want to crop from top, bottom, right, or left. This is where the Anchor name makes sense.